src/config/networks/polygon.ts

```diff
@@ -26,6 +26,9 @@ const polygon: NetworkConfig = {
     staging: {
```

Comment:
I would add staging links as baseConfig, as if used like that, staging services would be used for all environments (PRs, staging, test builds....) and we would only use production services when we are creating release candidate or production build